defaults/config.js

```diff
@@ -139,6 +139,27 @@ module.exports = {
 	// This value is set to `2048` kilobytes by default.
 	prefetchMaxImageSize: 2048,
 
+	// ### `fileUpload`
+	//
+	// Allow uploading files to the server hosting The Lounge.
+	// If enabled, The Lounge will store files in the `${THELOUNGE_HOME}/uploads` folder.
+	//
+	// Files will not expire or be removed by The Lounge, which can cause issues
+	// depending on your hardware, for example in terms of disk usage.
+	//
+	// The available keys for the `fileUpload` object are:
+	//
+	// - `enable`: When `true`, the identd daemon runs on server start.
+	// - `maxFileSize`: When file upload is enabled, users sending files above this
+	//    limit will be prompted an error message in their browser.
+	//    Specified value is in kilobytes. Default value is 10000 kilobytes (10 MiB).
+	//
+	// `-1` will disable the file size limit and will allow files of any size. Use at your own risk.
+	fileUpload: {
+		enable: true,
+		maxFileSize: 10000,
```

Comment:
@xPaw, why did you change this from 10240 to 10000? Other fields (well, at least `prefetchMaxImageSize`) are a power of 2, and it's also more predictable for users as most OS show sizes in MB. I'd rather remember to check that a file is < 10MB and not 9.77MB :)

Reply:
I changed the multiplier to 1024 to match the other max size field we have in the config already.

Reply:
Right, but you made the change so that it multiplies 10000kb into 10000*1024 bytes, this value should still be 10240 so it's 10mb (10*1024*1024 = 10,485,760 bytes)... no?
At the very least, `10240` here would be consistent with the `2048` above.

Reply:
You're right, fixed.